components/MentorFormRegistration.js

```diff
@@ -203,6 +206,34 @@ export default function MentorForm() {
     document.querySelector(className).style.display = "none";
   }
 
+  function updateTime() {
+    var timezone = document.getElementById("timezone").value;
+    let bookSessionCopy = formData.bookSession[0];
+    bookSessionCopy.datePickerTimezone = timezone;
+    setFormData({
+      ...formData,
+      bookSession: [bookSessionCopy],
+    });
+  }
+
+  function updateDatePickerDay(e) {
+    let bookSessionCopy = formData.bookSession[0];
+    bookSessionCopy.datePickerDay = e.target.value.split(",");
+    setFormData({
+      ...formData,
+      bookSession: [bookSessionCopy],
+    });
+  }
+
+  function updateDatePickerTime(e) {
+    let bookSessionCopy = formData.bookSession[0];
+    bookSessionCopy.datePickerTime = e.target.value.split(",");
+    setFormData({
+      ...formData,
+      bookSession: [bookSessionCopy],
+    });
```

Comment:
we have to provide slot like... if he chooses 1-1 meeting as 30.
so we have to show all slots from 00:00 to 24:00 in slot of each 30min.
00:00-00:30 , ......